lib/eco/dotviewer/graphdisplay.py

```diff
@@ -47,14 +47,16 @@ def permute_mods(base, args):
 
 class Display(object):
 
-    def __init__(self, (w,h)=(800,680)):
+    def __init__(self, xxx_todo_changeme=(800,680)):
         # initialize the modules by hand, to avoid initializing too much
         # (e.g. the sound system)
+        (w,h) = xxx_todo_changeme
         pygame.display.init()
         pygame.font.init()
         self.resize((w,h))
 
-    def resize(self, (w,h)):
+    def resize(self, xxx_todo_changeme1):
```

Comment:
Variable names like this scare me!

Reply:
This is from the `2to3` conversion. I forgot to rename those variables again.